lib/pip.js

```diff
@@ -147,11 +153,29 @@ function installRequirements(
  */
 function dockerPathForWin(options, path) {
   if (process.platform === 'win32' && options.dockerizePip) {
-    return path.replace('\\', '/');
+    return path.replace(/\\/g, '/');
   }
   return path;
 }
 
+/**
+ * get commands to slimmen the installed requirements
+ * @param {Object} options
+ * @param {string} targetRequirementsFolder
+ * @return {Array}
+ */
+function getSlimPackageCommands(options, targetRequirementsFolder) {
+  const folderPath = dockerPathForWin(options, targetRequirementsFolder);
+  const stripCmd = [
+    `&& find ${folderPath} -name "*.so" -exec strip {} ;`,
+    `&& find ${folderPath} -name "*.py[c|o]" -exec rm -rf {} +`,
```

Comment:
Any reason to use `-exec rm` instead of find's `-delete` option?

Reply:
To be honest I have never used t go for `-delete` - it felt natural to go with `rm {} +` since it's always there and as far as I know `+` makes it not span the redundant processes :thinking: 

Thanks for pointing it out - I have to consider opting for `-delete`!   

It is available at the `alpine` docker image sooo, maybe I should run some experiments to check which performs better? 
Do you prefer `-delete` usually :slightly_smiling_face: ?